app/lib/dns.server.ts

```diff
@@ -190,11 +190,22 @@ export const deleteRecord = async (
 ) => {
   try {
     if (!isNameValid(name, username)) {
-      throw new Error('Invalid name provided');
+      logger.error('DNS Error in deleteRecord - invalid record name provided', {
+        name,
+        username,
+        baseDomain: buildUserBaseDomain(username),
+      });
+      throw new Error('DNS Error in upsertRecord - invalid name provided');
```

Comment:
Shouldn't this be `deleteRecord` instead of `upsertRecord`?

Reply:
Yes, fixing...